pages/common/jpegoptim.md

```diff
@@ -0,0 +1,15 @@
+# jpegoptim
+
+> Optimise jpeg images.
+
+- Optimise a set of jpeg images:
+
+`jpegoptim {{image1.jpeg}} {{image2.jpeg}} {{imageN.jpeg}}`
+
+- Optimise jpeg images, stripping all non-essential information:
```

Comment:
Yea, I think we should clarify what sort of optimisations are applied by default, and what does `--strip-all` do.